examples/mongodb/1.8/initialize-mongo-single.js

```diff
@@ -0,0 +1,83 @@
+let error = true
+
+let res = [
+// remove if exists
+ db = db.getSiblingDB('inventory'),
+ db.customers.drop(),
+ db.orders.drop(),
+ db.products.drop(),
+
+  rs.initiate({
+    _id: "rs0",
+    members: [ { _id: 0, host: "localhost:27017" } ]
```

Comment:
not sure  setting host to  `localhost"` is correct? previous shell script was reading defining HOST it differently. 
maybe we need to use `getHostName()`

Reply:
@ismailsimsek first, thank you again for taking over that work! See my other comment about `rs.initiate` below.